### DISCUSSION TITLE
Recommend me some good web developer teachers

### DISCUSSION THREAD

**Discussion by Dopplenum** [community/community#118198] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I just want the names of the good teachers

**Comment by Sohailkhan2k02** [comment, marked as answer]
Harkirat
Love Babbar
Chai with code

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Dopplenum !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!

**Comment by Dopplenum** [comment]
Ok as I am new to this so it is of great help

**Comment by jfullstackdev** [comment]
you can still post questions in here to be answered, are you new to coding journey ? I'm not promoting GitHub Copilot but it's really worth it though it's not free for most of us, AI is not a gimmick at all, it can act as your teacher for coding questions, it can even document the code itself
